engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -97,7 +97,7 @@
     private LocalPlayer localPlayer;
     @In
     private WorldProvider worldProvider;
-    private Camera playerCamera;
+    public Camera playerCamera;
```

Comment:
From the PR alone it seems unnecessary. If it's needed elsewhere it's good practice to comment your own PR about changes that might be puzzling. :wink: